src/toBeReviewed.js

```diff
@@ -0,0 +1,26 @@
+var animalData = [
+  {
+    id: 1,
+    title: "hippo",
+    faveFood: "carrots",
+  },
+  {
+    id: 2,
+    title: "Cat",
+    faveFood: "carrots",
+  },
+  {
+    id: 3,
+    title: "ducks",
+    faveFood: "breadcrumbs",
+  },
+];
+
+exports.findAnimal = function () {
+  for (x = 0; x < 3; x++) {
```

Comment:
Explicitly declare the variable in for loop using "let" for a well-defined lexical scope.